src/public/meeting.ts

```diff
@@ -325,4 +349,22 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel);
     sendMessageToParent('meeting.getAppContentStageSharingState', callback);
   }
+
+  /**
+   * Registers a handler for changes to speaking array.
+   * @param handler The handler to invoke when the array of participants speaking changes. A value
+   * of 0 (no participants speaking) or 1 (one or more participants are speaking) is provided.
+   * @param includeParticipants is a flag to opt-in to receive participant ids of who is speaking.
+   * Default value is false.
+   */
+  export function registerDetectSpeakingStateChangedHandler(
+    handler: (isSpeakingDetected: ISpeakingState) => void,
+    includeParticipants?: boolean,
```

Comment:
Not sure if matters, but is it more common to just do includeParticipants = false to make it more implicit what the default value is? I'm not sure I've ever seen an optional Boolean before, but both ways should work.

Reply:
@ryanbliss to follow up on this part, I believe it's more common to handle the default false value on the teams side. For example, in location.ts, there is an optional boolean to show a map: https://github.com/OfficeDev/microsoft-teams-library-js/blob/a3ae40a9606803c68e280a022fe9849b66901e99/src/public/location.ts#L21 . Then on the teams side there is some logic in the hook file that handles this API for the default false value. From what I could find, I didn't see any example in the SDK where we set default values on this side.

Reply:
@arocc good to know, thank you!